packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -416,51 +461,68 @@ const BarfTurnTasks: GarboTask[] = [
       sobriety: "sober",
     },
   ),
-  {
-    name: "Map for Pills",
-    ready: () =>
-      globalOptions.ascend &&
-      clamp(myAdventures() - digitizedMonstersRemaining(), 1, myAdventures()) <=
-        availableAmount($item`Map to Safety Shelter Grimace Prime`),
-    completed: () => false,
-    do: () => {
-      const choiceToSet =
-        availableAmount($item`distention pill`) <
-        availableAmount($item`synthetic dog hair pill`) +
-          availableAmount($item`Map to Safety Shelter Grimace Prime`)
-          ? 1
-          : 2;
-      setChoice(536, choiceToSet);
-      ensureEffect($effect`Transpondent`);
-      use($item`Map to Safety Shelter Grimace Prime`);
-      return true;
-    },
-    spendsTurn: true,
-    sobriety: "drunk",
-  },
-  {
-    name: "Barf",
-    completed: () => myAdventures() === 0,
-    outfit: () => {
-      const lubing = get("dinseyRollercoasterNext") && have($item`lube-shoes`);
-      return barfOutfit(lubing ? { equip: $items`lube-shoes` } : {});
-    },
-    do: () => $location`Barf Mountain`,
-    combat: new GarboStrategy(
-      () => Macro.meatKill(),
-      () =>
-        Macro.if_(
-          `(monsterid ${embezzler.id}) && !gotjump && !(pastround 2)`,
-          Macro.meatKill(),
-        ).abort(),
-    ),
-    post: () => completeBarfQuest(),
-    spendsTurn: true,
-  },
 ];
 
+function nonBarfTurns(): number {
+  const sobriety = sober() ? "sober" : "drunk";
+  return sum(
+    NonBarfTurnTasks.filter(
+      (t) => (undelay(t.sobriety) ?? sobriety) === sobriety,
+    ),
+    (t) => undelay(t.turns),
+  );
+}
+
+export const TurnGenQuest: Quest<GarboTask> = {
+  name: "Turn Gen",
+  tasks: TurnGenTasks,
+};
+
+export const WandererQuest: Quest<GarboTask> = {
+  name: "Wanderers",
+  tasks: BarfTurnTasks,
+  completed: () => !canContinue(),
+};
+
+export const NonBarfTurnQuest: Quest<GarboTask> = {
+  name: "Non Barf Turn",
+  tasks: NonBarfTurnTasks,
+  completed: () =>
+    !canContinue() &&
+    clamp(myAdventures() - digitizedMonstersRemaining(), 1, myAdventures()) >=
+      nonBarfTurns(),
+};
+
 export const BarfTurnQuest: Quest<GarboTask> = {
   name: "Barf Turn",
-  tasks: BarfTurnTasks,
+  tasks: [
+    {
+      name: "Barf",
+      completed: () => myAdventures() === 0,
+      outfit: () => {
+        const lubing =
+          get("dinseyRollercoasterNext") && have($item`lube-shoes`);
+        return barfOutfit(lubing ? { equip: $items`lube-shoes` } : {});
+      },
+      do: () => $location`Barf Mountain`,
```

Comment:
```suggestion
      do: $location`Barf Mountain`,
```
not sure if this is new, but I do think we prefer this